src/libs/Navigation/AppNavigator/MainDrawerNavigator.js

```diff
@@ -49,7 +49,7 @@ const MainDrawerNavigator = (props) => {
 
     // Wait until reports are fetched and there is a reportID in initialParams
     if (!initialParams.reportID) {
-        return <FullScreenLoadingIndicator />;
+        return <FullScreenLoadingIndicator name="Main Drawer Loader" timeout={15 * 1000} />;
```

Comment:
**suggestion:** Move the `15 * 1000` into a `CONST.TIMING.SPINNER_TIMEOUT` variable